be/src/connector/file_connector.cpp

```diff
@@ -0,0 +1,160 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present,
+// StarRocks Limited.
+
+#include "connector/file_connector.h"
+
+#include "column/chunk.h"
+#include "exec/exec_node.h"
+#include "exec/vectorized/csv_scanner.h"
+#include "exec/vectorized/json_scanner.h"
+#include "exec/vectorized/orc_scanner.h"
+#include "exec/vectorized/parquet_scanner.h"
+#include "exprs/expr.h"
+
+namespace starrocks {
+namespace connector {
+using namespace vectorized;
+
+// ================================
```

Comment:
maybe these separators are not necessary?

Reply:
I love those separators, it's helpful to readers & writers.